packages/cli/test/integration/saml/saml.api.test.ts

```diff
@@ -266,89 +260,3 @@ describe('Check endpoint permissions', () => {
 		});
 	});
 });
-
-describe('SAML login flow', () => {
-	beforeEach(async () => {
-		await enableSaml(true);
-	});
-
-	test('should trigger onUserLoginSuccess hook', async () => {
-		const mockedHandleSamlLogin = jest.spyOn(Container.get(SamlService), 'handleSamlLogin');
-
-		mockedHandleSamlLogin.mockImplementation(
-			async (): Promise<{
-				authenticatedUser: User;
-				attributes: SamlUserAttributes;
-				onboardingRequired: false;
-			}> => {
-				return {
-					authenticatedUser: someUser,
-					attributes: {
-						email: someUser.email,
-						firstName: someUser.firstName,
-						lastName: someUser.lastName,
-						userPrincipalName: someUser.email,
-					},
-					onboardingRequired: false,
-				};
-			},
-		);
-
-		const mockedHookOnUserLoginSuccess = jest.spyOn(
-			Container.get(InternalHooks),
-			'onUserLoginSuccess',
-		);
-		mockedHookOnUserLoginSuccess.mockImplementation(
-			async (userLoginData: { user: User; authenticationMethod: AuthenticationMethod }) => {
-				expect(userLoginData.authenticationMethod).toEqual('saml');
-				return;
-			},
-		);
-		await authOwnerAgent.post('/sso/saml/acs').expect(302);
-		expect(mockedHookOnUserLoginSuccess).toBeCalled();
```

Comment:
Should we instead check that the event was emitted?

Reply:
Yeah that would be ideal. Was just wondering why the entire test case was deleted?

Reply:
These two tests are validating calls for hooks that no longer exist:

> Internal hooks that existed only for the event bus have been deleted, as they are now covered by events: `onNodeBeforeExecute`, `onNodePostExecute`, `onWorkflowBeforeExecute`, `onUserLoginSuccess`, `onUserLoginFailed`